TOC.md

```diff
@@ -5,3 +5,16 @@
 
 ## Chunks
 - [Terrain Generation Is Random](Chunks/terrain-generation-is-random.md)
+
+## Player Model
```

Comment:
for some erason now there are 2 player model tests that are the same. Could you remove the extra one?

Reply:
That would mean removing the PlayerModel folder